v-next/hardhat-ignition-ethers/test/deploy-with-ethers-result.ts

```diff
@@ -79,16 +85,18 @@ describe("deploy with ethers result", () => {
       return { foo, bar };
     });
 
-    const result = await this.connection.ignition.deploy(moduleDefinition);
+    const connection = await createConnection();
+
+    const result = await connection.ignition.deploy(moduleDefinition);
 
-    assert.isTrue(await result.foo.isFoo());
-    assert.isTrue(await result.bar.isBar());
+    assert.equal(await result.foo.isFoo(), true);
+    assert.equal(await result.bar.isBar(), true);
 
     // A function on the abi will not be defined on the ethers contract,
-    // but more importantly this should how up as a type error.
+    // but more importantly should how up as a type error.
 
     // TODO: add @ts-expect-error when we have typescript support
```

Comment:
Is this TODO still relevant?

Reply:
yes, this refers to potential typechain support in the future which is not yet implemented